packages/e2e-tests/specs/performance/site-editor.test.js

```diff
@@ -75,112 +77,121 @@ describe( 'Site Editor Performance', () => {
 		}, html );
 		await saveDraft();
 
-		id = await page.evaluate( () =>
+		postId = await page.evaluate( () =>
 			new URL( document.location ).searchParams.get( 'post' )
 		);
 	} );
 
 	afterAll( async () => {
+		const resultsFilename = basename( __filename, '.js' ) + '.results.json';
+		writeFileSync(
+			join( __dirname, resultsFilename ),
+			JSON.stringify( results, null, 2 )
+		);
+
 		await deleteAllTemplates( 'wp_template' );
 		await deleteAllTemplates( 'wp_template_part' );
 		await activateTheme( 'twentytwentyone' );
 	} );
 
-	beforeEach( async () => {
-		await visitSiteEditor( {
-			postId: id,
-			postType: 'page',
-		} );
-	} );
-
-	it( 'Loading', async () => {
-		const editorURL = await page.url();
-
-		// Number of sample measurements to take.
+	describe( 'Loading', () => {
+		// Number of measurements to take.
 		const samples = 3;
 		// Number of throwaway measurements to perform before recording samples.
-		// Having at least one helps ensure that caching quirks don't manifest in
-		// the results.
+		// Having at least one helps ensure that caching quirks don't manifest
+		// in the results.
 		const throwaway = 1;
+		const iterations = Array.from(
+			{ length: samples + throwaway },
+			( _, i ) => i + 1
+		);
```

Comment:
gotta admit this jumps out. I understand what we're doing, but would it be okay to move this into a function with a standard name? something like `range` or `sequence`?

```js
const range = n => Array.from( … )

const iterations = range( throwaway + samples );
```

Reply:
https://github.com/WordPress/gutenberg/pull/48138/commits/12c0558a35d42270101a357fbf2c8faacec3d56f